crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1549,16 +1556,14 @@ where
 
                         client
                             .handle_certificate_v3(req, client_addr)
-                            .instrument(
-                                tracing::trace_span!("handle_certificate", authority =? name.concise()),
-                            )
+                            .monitor_cancellation()
+                            .instrument(trace_span!("handle_certificate_v3", cancelled = false, authority =? concise_name))
                             .await
                     } else {
                         client
                             .handle_certificate_v2(request_ref.certificate, client_addr)
-                            .instrument(
-                                tracing::trace_span!("handle_certificate", authority =? name.concise()),
-                            )
+                            .monitor_cancellation()
+                            .instrument(trace_span!("handle_certificate_v2", cancelled = false, authority =? concise_name))
```

Comment:
to my knowledge these won't be dropped. why do we wrap them with `monitor_cancellation` here?

Reply:
good point!